### DISCUSSION TITLE
[Hacktoberfest'22] Revamped Peerlist `README` Badge Generator

### DISCUSSION THREAD

**Discussion by vinitshahdeo** [vinitshahdeo/peerlist-readme-badge#30] [General]
Hello 👋 

I'm opening this thread to discuss the dev work for the revamped Peerlist `README` Badge Generator. My gratitude to @TejasviArora for helping me with the new design.

![Peerlist README Badge Generator](https://user-images.githubusercontent.com/20594326/194946605-2aafd40e-1eaa-43f7-9b35-0cb45bd1332a.png)

- You can see the older view here: [peerlist-readme-badge.netlify.app](https://peerlist-readme-badge.netlify.app/)
- Here's the [Figma mock](https://www.figma.com/proto/DGSis1DiZrULEiWydKtpJb/Peerlist?page-id=0%3A1&node-id=2%3A2&viewport=476%2C632%2C0.38&scaling=scale-down-width) for the revamped website.
- View the [API Documentation](https://www.postman.com/restless-rocket-22186/workspace/peerlist-readme-badges/collection/6178851-67cf0bab-e978-4a37-b3ad-a5b3b42bf69e) on Postman.
- Refer to [`README.md`](https://github.com/vinitshahdeo/peerlist-readme-badge#readme) to learn more.

[![Peerlist Badges Postman Collection](http://img.shields.io/badge/Postman-Collection-orange.svg?style=flat&logo=postman)](https://www.postman.com/restless-rocket-22186/workspace/peerlist-readme-badges/collection/6178851-67cf0bab-e978-4a37-b3ad-a5b3b42bf69e) [![Visual Studio](https://badgen.net/badge/icon/visualstudio?icon=visualstudio&label)](https://github.dev/vinitshahdeo/peerlist-readme-badge/) ![GitHub Hacktoberfest combined status](https://img.shields.io/github/hacktoberfest/2022/vinitshahdeo/peerlist-readme-badge?logo=github&logoColor=white)

Interested members, please chime in! We'll create issues and start dev work by Friday. If needed, we will have a quick sync-up on G-Meet. 

> Your feedback is highly appreciated.

Thank you!

[![Peerlist](https://peerlist-readme-badge.herokuapp.com/api/vinitshahdeo)](https://peerlist.io/vinitshahdeo)

**Comment by raibove** [comment]
Hi!
I would like to join the discussion and contribute to this issue

**Comment by rajatsonibkn2002** [comment]
Hey!
I would love to contribute to the project.

**Comment by Shivank12batra** [comment]
Hey Vinit! I would love to contribute to the project as well.

**Comment by vinitshahdeo** [comment]
Hey @raibove, @rajatsonibkn2002, and @Shivank12batra - thanks for showing your interest. As next steps, please submit your details here - https://forms.gle/bt9hA2giHC8qUpzE7

> Meeting invites are on the way!

**Happy Hacktoberfest'22!**

**Comment by prathamesh-dukare** [comment]
Hey I'd love to be the part of this new improvements

**Comment by vinitshahdeo** [comment]
Thanks for showing your interest. You must've received a calendar invite by now. Please let me know if you aren't available.

#### **📆  Saturday, October 15⋅10:00 – 10:30pm**

Hope to see you there! ✨

**Comment by Ayan-thecodeking** [comment]
Hey @vinitshahdeo I'm also in .. pls send me too

**Reply by vinitshahdeo** [reply]
Sure - please provide your details here: https://forms.gle/bt9hA2giHC8qUpzE7

**Comment by badger751** [comment]
hi!
i would like to join the discussion and contribute to this issue
